common/app/views/fragments/amp/stylesheets/liveBlog.scala.html

```diff
@@ -78,11 +78,11 @@
 }
 
 .liveblog-block-byline__img {
-    display: inline;
-    border-radius: 50%;
-    max-width: 2.25rem;
     vertical-align: middle;
-    margin-right: 0.25rem;
+}
+
+.liveblog-block-byline__img img {
```

Comment:
I could not find a way to style this element directly without nesting. We could always dispense with the rounded borders for the contributor image in AMP, I guess. The only classes we can hook into are generic classes provided by AMP.